client/ayon_usd/__init__.py

```diff
@@ -1,4 +1,15 @@
 """USD Addon for AYON - client part."""
+# flake8: noqa F401
+
+# TODO ayon_api has a check for headless mode. we need to see if that system covers all fronts 
+HEADLESS_MODE = True 
```

Comment:
This is how it is used in ayon-launcher:

```suggestion
HEADLESS_MODE_ENABLED = os.getenv("AYON_HEADLESS_MODE") == "1"
```